bridges/snowbridge/pallets/ethereum-client/benchmark.md

```diff
@@ -15,25 +17,31 @@ is the main bottleneck. Test data
 is real from goerli network which contains 512 public keys from sync committee.
 
 ## sync_committee_period_update
-Base line benchmark for extrinsic [sync_committee_period_update](https://github.com/Snowfork/snowbridge/blob/8891ca3cdcf2e04d8118c206588c956541ae4710/parachain/pallets/ethereum-client/src/lib.rs#L233)
+
+Base line benchmark for
+extrinsic [sync_committee_period_update](https://github.com/Snowfork/snowbridge/blob/8891ca3cdcf2e04d8118c206588c956541ae4710/parachain/pallets/ethereum-client/src/lib.rs#L233)
 
 ## bls_fast_aggregate_verify
+
 Subfunction of extrinsic `sync_committee_period_update` which does what
 [FastAggregateVerify](https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-bls-signature-04#section-3.3.4) requires.
 
 ## bls_aggregate_pubkey
+
 Subfunction of `bls_fast_aggregate_verify` which decompress and instantiate G1 pubkeys only.
 
 ## bls_verify_message
-Subfunction of `bls_fast_aggregate_verify` which verify the prepared signature only.
 
+Subfunction of `bls_fast_aggregate_verify` which verify the prepared signature only.
 
 # Result
 
 ## hardware spec
+
 Run benchmark in a EC2 instance
+
 ```
-cargo run --release --bin polkadot-parachain --features runtime-benchmarks -- benchmark machine --base-path /mnt/scratch/benchmark
+cargo run --release --bin polkadot-parachain-omni-node --features runtime-benchmarks -- benchmark machine --base-path /mnt/scratch/benchmark
```

Comment:
From my understanding, all `node benchmark` commands should eventually be moved to `omni-bencher` CLI. I am not sure if the tool is ready yet, but good to know what is the end goal.